turbopack/crates/turbo-tasks/src/manager.rs

```diff
@@ -335,9 +335,13 @@ struct CurrentTaskState {
     /// This is taken (and becomes `None`) during teardown of a task.
     cell_counters: Option<AutoMap<ValueTypeId, u32, BuildHasherDefault<FxHasher>, 8>>,
 
-    /// Cells for locally allocated Vcs (`RawVc::LocalCell`). This is freed
-    /// (along with `CurrentTaskState`) when the task finishes executing.
+    /// Cells for locally allocated Vcs (`RawVc::LocalCell`). This is freed (along with
+    /// `CurrentGlobalTaskState`) when the task finishes executing.
     local_cells: Vec<TypedSharedReference>,
+
+    /// Tracks currently running local tasks, and defers cleanup of the global task until those
+    /// complete.
+    local_task_tracker: TaskTracker,
```

Comment:
Will this affect memory usage?

Reply:
I've added memory benchmarks to the PR summary. There is no meaningful/measurable change in memory consumption.